tests/Jira/ApiTest.php

```diff
@@ -149,6 +149,40 @@ public function testFindVersionByName()
 		);
 	}
 
+	public function testGetResolutions()
+	{
+		$response = '[{"self":"https://test.atlassian.net/rest/api/2/resolution/1","id":"1","description":"A fix for this issue is checked into the tree and tested.","name":"Fixed"},{"self":"https://test.atlassian.net/rest/api/2/resolution/10000","id":"10000","description":"This issue won\'t be actioned.","name":"Won\'t Do"}]';
```

Comment:
Looks almost good:

1.  why you've called file `api_status.json` while it is returning resolutions, not statuses?
2. maybe we should place `resources` folder right in `/tests/` and not based on test class location?